x/concentrated-liquidity/types/msgs_test.go

```diff
@@ -285,3 +288,53 @@ func TestMsgWithdrawPosition(t *testing.T) {
 		}
 	}
 }
+
+func TestConcentratedLiquiditySerialization(t *testing.T) {
+	pk1 := ed25519.GenPrivKey().PubKey()
+	addr1 := sdk.AccAddress(pk1.Address()).String()
+	defaultTickSpacing := uint64(1)
+	defaultPoolId := uint64(1)
+
+	testCases := []struct {
+		name  string
+		clMsg sdk.Msg
+	}{
+		{
+			name: "MsgCreateConcentratedPool",
+			clMsg: &model.MsgCreateConcentratedPool{
+				Sender:      addr1,
+				Denom0:      "foo",
+				Denom1:      "bar",
+				TickSpacing: defaultTickSpacing,
+			},
+		},
+		{
+			name: "MsgWithdrawPosition",
+			clMsg: &types.MsgWithdrawPosition{
+				PoolId:          defaultPoolId,
+				Sender:          addr1,
+				LowerTick:       int64(10000),
+				UpperTick:       int64(20000),
+				LiquidityAmount: sdk.NewInt(100),
+			},
+		},
+		{
+			name: "MsgCreatePosition",
+			clMsg: &types.MsgCreatePosition{
+				PoolId:          defaultPoolId,
+				Sender:          addr1,
+				LowerTick:       int64(10000),
+				UpperTick:       int64(20000),
+				TokenDesired0:   sdk.NewCoin("foo", sdk.NewInt(1000)),
+				TokenDesired1:   sdk.NewCoin("bar", sdk.NewInt(1000)),
+				TokenMinAmount0: sdk.OneInt(),
+				TokenMinAmount1: sdk.OneInt(),
+			},
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			apptesting.TestMessageConcentratedLiquiditySerialization(t, tc.clMsg)
```

Comment:
I might be messing something but why is not possible to reuse the existing helper like here:
https://github.com/osmosis-labs/osmosis/blob/f611e01c7fb9ab9d5348250cb0b65e08f3ec651b/x/swaprouter/types/msgs_test.go#L323

Reply:
@p0mvn oh, I completely misunderstood the task at the beginning, now I get what is required, I will change the code and remove stuff I wrote to apptesting

Reply:
No worries. Please tag me once this is ready for another round of review